web/forms.py

```
@@ -1802,3 +1804,70 @@ class StudyGroupForm(forms.ModelForm):
+    )
+
+    work_file = forms.FileField(
+        help_text="Upload your document (file type and size restrictions apply)",
+        widget=forms.FileInput(),
+    )
+
+    class Meta:
+        model = WorkSubmission
+        fields = ["title", "work_type", "subject", "assignment", "due_date", "description", "work_file"]
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)